legend-depot-artifacts-purge/src/main/java/org/finos/legend/depot/store/artifacts/purge/services/ArtifactsPurgeServiceImpl.java

```diff
@@ -124,15 +150,14 @@ private MetadataEventResponse deleteOldestProjectVersions(ProjectData project, i
                     delete(project.getGroupId(), project.getArtifactId(), versionId.toVersionIdString());
                     versionIds.remove(versionId);
                     project.removeVersion(versionId.toVersionIdString());
-                    response.addMessage(String.format("%s-%s-%s purge", project.getGroupId(), project.getArtifactId(), versionId.toVersionIdString()));
-                    PrometheusMetricsFactory.getInstance().incrementCount(VERSION_PURGE_COUNTER);
```

Comment:
removed the increment in counter here as we already do this when we do delete in line 150